File: Frontend-POS/RBMS-POS-Client/src/app/core/api/models/customer-menu-categories-response-model.ts
/* tslint:disable */
/* eslint-disable */
import { CustomerCategoryModel } from '../models/customer-category-model';
import { CustomerSubCategoryModel } from '../models/customer-sub-category-model';
export interface CustomerMenuCategoriesResponseModel {
  categories?: Array<CustomerCategoryModel> | null;
  subCategories?: Array<CustomerSubCategoryModel> | null;
}

File: Backend-POS/POS.Main/POS.Main.Business.Payment/Models/SelfOrder/CustomerMenuCategoriesResponseModel.cs
namespace POS.Main.Business.Payment.Models.SelfOrder;

public class CustomerMenuCategoriesResponseModel
{
    public List<CustomerCategoryModel> Categories { get; set; } = new();
    public List<CustomerSubCategoryModel> SubCategories { get; set; } = new();
}

public class CustomerCategoryModel
{
    public int CategoryType { get; set; }
    public string Name { get; set; } = string.Empty;
}

public class CustomerSubCategoryModel
{
    public int SubCategoryId { get; set; }
    public int CategoryType { get; set; }
    public string Name { get; set; } = string.Empty;
}
